notebooks/tel_and_site/subsys_req_ver/mthexapod/SITCOM-2190_apply_fft_to_motor_current.ipynb

```diff
@@ -0,0 +1,743 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    signal_threshold = 0.5
</pre><p>The comment says 0.75 while you use 0.5</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>